wandb/sdk/wandb_run.py

```diff
@@ -2041,8 +2041,8 @@ def _log_job(self) -> None:
         )
 
         for job_creation_function in [
-            self._create_repo_job,
             self._create_artifact_job,
+            self._create_repo_job,
```

Comment:
Lets keep the preference for repo -> artifact -> container

Reply:
Sorry to push back here, but why? I feel like artifacts are something we maintain/own versus a github repo. Is there more customers that use the repo vs artifacts?